File: src/1318-Minimum-Flips-to-Make-a-OR-b-Equal-to-c/1318.java
class Solution {
    public int minFlips(int a, int b, int c) {
        int res = 0;
        while (a > 0 || b > 0 || c > 0) {
            if ((c & 1) == 1) res += ((a & 1) == 1) || ((b & 1) == 1) ? 0 : 1;
            else res += (a & 1) + (b & 1);
            a >>= 1; b >>= 1; c >>= 1;
        }
        return res;
    }
}

File: src/1318-Minimum-Flips-to-Make-a-OR-b-Equal-to-c/1318.js
var minFlips = function(a, b, c) {
    let res = 0;
    while (a || b || c) {
        if (c & 1) res += (a & 1) || (b & 1) ? 0 : 1;
        else res += (a & 1) + (b & 1);
        a >>= 1, b >>= 1, c >>= 1;
    }
    return res;
};